BUILD.bazel

```diff
@@ -244,6 +247,7 @@ orfs_flow(
             "CORE_UTILIZATION": "20",
             "MACRO_PLACE_HALO": "20 20",
             "RTLMP_FLOW": "1",
+	    "BLOCKS": boom_tile_blocks,
```

Comment:
This breaks ORFS. "BLOCKS" is a Makefile specific construct, don't use it from ORFS.

bazel-orfs sets "MACROS"

https://github.com/The-OpenROAD-Project/bazel-orfs/blob/767676d644d07a459393c010699e92551ffaec5b/openroad.bzl#L386-L388